faq/blockstorage.mdx

```diff
@@ -5,12 +5,12 @@ meta:
 content:
   h1: Block Storage
   paragraph: Discover the difference between file storage and Block Storage and get answers to your performance and security concerns. 
-image: blockstorage.webp
+hero: assets/blockstorage.webp
 ---
 
 ## Why should I use Block Storage instead of local storage?
 
-The size of local storage depends on your Instance type. With today’s configurations, the maximum local storage available is 600 GB.
+The size of local storage depends on your Instance type. With today's configurations, the maximum local storage available is 600 GB.
 
 Block storage allows you to create bigger volumes and those volumes are persistent which means that they can live without being attached to an Instance.
 
```

Comment:
Table is broken below

![Capture d’écran 2022-10-19 à 9 37 23 AM](https://user-images.githubusercontent.com/341803/196627503-1d5449f7-6312-4661-8114-5b4a22bc67ef.png)